addon/index.js

```diff
@@ -538,20 +558,13 @@ export function changeset(obj, validateFn = defaultValidatorFn, validationMap =
         this._deleteKey(ERRORS, key);
 
         if (!isEqual(oldValue, value)) {
-          this._recursivelySet(key, value, changes);
+          deepSet(changes, key, value);
         } else if (key in changes) {
           delete changes[key];
         }
-
         this.notifyPropertyChange(CHANGES);
         this.notifyPropertyChange(root);
```

Comment:
i used to have it where both the root and they key got triggered, but triggering both is redundant, and any child keys get triggered from the relay.